scripts/create_conda_packages.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © 2016, Continuum Analytics, Inc. All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+""""""
+
+from __future__ import print_function
+
+# Standard library imports
+import argparse
+import ast
+import os
+import subprocess
+import shutil
+import sys
+
+# Constants
+HERE = os.path.abspath(os.path.dirname(__file__))
+ROOT = os.path.dirname(HERE)
+PYTHON_VERSIONS = ('2.7', '3.5', '3.6')
+
+
+def get_version(module='anaconda_project'):
+    """Get version."""
+    with open(os.path.join(ROOT, module, 'version.py'), 'r') as f:
+        data = f.read()
+    lines = data.split('\n')
+    for line in lines:
+        if line.startswith('VERSION_INFO'):
+            version_tuple = ast.literal_eval(line.split('=')[-1].strip())
+            version = '.'.join(map(str, version_tuple))
+            break
+    return version
+
+
+VERSION = get_version()
```

Comment:
Why you doing this and not simply loading it as a module?

Reply:
To avoid loading the module. Not sure why but 8 have always donde packages like this

Reply:
I may be missing something... but it be good to actually know why

Reply:
I remember now, in some cases when you import the module itself, you might also be importing something from a dependency, which may or may not exist in the setup the user has prior to installing things, so that is why we don't directly import the module but instead we parse the version with vanilla python. This might not affect this library, but I think is in gneral good practice.